app/uk/gov/hmrc/eoricommoncomponent/frontend/views/ServiceName.scala

```diff
@@ -42,6 +42,6 @@ object ServiceName {
     shortName(service)
 
   def service(implicit request: Request[_]): Service =
-    Service.serviceFromRequest.getOrElse(Service.NullService)
+    Service.serviceFromRequest.getOrElse(Service("", ""))
```

Comment:
We might not need the "null" service any more.  We can see if we can just throw an exception here.

Reply:
Yes, it just to compile the service and be able to present something working

I think that CDS Service because it's required for enrolment checks we would like to keep somewhere as an object

Reply:
What I mean is that if we can't work out the "Service" from the url then maybe that is an error.   I.e "getOrElse(throw Exception)

Reply:
I answered in wrong place. I meant the comment below. Yeah, here we can throw an exception

Reply:
Replacing empty service with exception will cause 955 TESTS FAILED

I'll leave it for now. I'm sure that tests probably will require fix in one place, but will check this at the end